tools/cargo-miden/src/build.rs

```diff
@@ -36,7 +36,7 @@ pub fn build_masm(
     }]);
     let project_type = if is_bin { "--exe" } else { "--lib" };
     let source_manager = Arc::new(DefaultSourceManager::default());
-    let options = midenc_compile::CompilerOptions::parse_options(&[project_type])
+    let options = midenc_compile::CompilerOptions::parse_options(&[project_type, "-l", "std"])
```

Comment:
I'm planning to make `-lstd` implicit, but this is a good workaround for the moment.